src/app/autofocus/autofocus.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="page-header" style="padding-bottom: 0;">
+    <h3>Autofocus</h3>
+
+    <p>This directive auto-focuses on a focusable form element on page load.</p>
```

Comment:
This directive auto-focuses on a focusable ~~form~~ element ~~on page load~~ when that element is rendered.